AGENTS.md

```diff
@@ -0,0 +1,71 @@
+# LangChain.js
+
+LangChain.js is a framework and ecosystem for building context-aware, LLM-powered applications with composable components, stateful agents, and production tooling.
+
+## Overview
+
+- Provides composable building blocks, chains, agents, and LangGraph tooling for LLM applications targeting Node.js, edge runtimes, browsers, and Deno
+- Monorepo managed with pnpm and TurboRepo; key workspaces include `@langchain/core` (base abstractions), `langchain` (chains/agents), and numerous provider packages under `libs/providers/` and `libs/langchain-community/`
+- Source lives across `langchain-core/`, `langchain/`, and `libs/*`; examples and notebooks are in `examples/` and `cookbook/`
+
+## Key Technologies and Frameworks
+
+- TypeScript targeting Node.js v20+ (repo `.nvmrc` pins v24.6.0)
+- pnpm workspaces (`pnpm-workspace.yaml`) with TurboRepo task orchestration (`turbo.json`)
+- Vitest and Jest (via Turbo) for unit and integration testing
```

Comment:
We are getting rid of Jest (it is still in some packages but we are in the process of changing that.)
```suggestion
- Vitest for unit and integration testing
```